tests/expressions/tests.py

```diff
@@ -332,12 +336,195 @@ def test_object_update_unsaved_objects(self):
         with self.assertRaisesMessage(ValueError, msg):
             acme.save()
 
+    def test_ticket_22288_in_f_integer(self):
```

Comment:
Usually we prefer for test names to describe the functionality, not reference the ticket. git blame can always be used to get back to the ticket if needed.